### PR TITLE
Add Arch AUR installation instructions

### PR DESCRIPTION
**Pull request by yitzchak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,4 +34,25 @@ that's built into gxi, so please don't use `cargo` to install gxi, as that won't
...
+### Installation on Arch/Manjaro
+
+The package for Arch Linux is [gxi-git](https://aur.archlinux.org/packages/gxi-git/). Building and installing
+(including dependencies) can be accomplished with:
+
+```sh
+yaourt -Sy gxi-git
+```
+
+Alternatively use ``makepkg``:
```
Comment (Cogitri) [inline README.md]: Although not a blocker, why have two '`' here?

File: README.md [hunk 2]
```diff
@@ -34,4 +34,25 @@ that's built into gxi, so please don't use `cargo` to install gxi, as that won't
 from a different rev can lead to very weird bugs.
 
 
-After these steps you should be able to run gxi simply by invoking `gxi`
\ No newline at end of file
+After these steps you should be able to run gxi simply by invoking `gxi`
+
+### Installation on Arch/Manjaro
+
+The package for Arch Linux is [gxi-git](https://aur.archlinux.org/packages/gxi-git/). Building and installing
```
Comment (Cogitri) [inline README.md]: Hm, that sounds like gxi is in the main repos already. Maybe something like ```gxi can be [found in the AUR](link)``` would be better?

File: README.md (hunk 1 above)
Comment (yitzchak) [reply README.md]: Just a typo, fixed.

File: README.md (hunk 2 above)
Comment (yitzchak) [reply README.md]: Good point.

**Comment by yitzchak** [comment]
I made some changes as requested, but please check my language again. I added a versioned package to the AUR also so I had to change the instructions a bit.

File: README.md (hunk 1 above)
Comment (Cogitri) [reply README.md]: Thanks :)

File: README.md [hunk 3]
```diff
@@ -34,4 +34,32 @@ that's built into gxi, so please don't use `cargo` to install gxi, as that won't
...
 
-After these steps you should be able to run gxi simply by invoking `gxi`
\ No newline at end of file
+After these steps you should be able to run gxi simply by invoking `gxi`
+
+### Installation on Arch/Manjaro
+
+There are two packages for gxi in Arch Linux's
+[AUR](https://aur.archlinux.org/). The first is the versioned package
+[gxi](https://aur.archlinux.org/packages/gxi/) and the second is the git
```
Comment (Cogitri) [inline README.md]: Maybe "The first one follows gxi's release cycle"? Dunno, I'm fine with the first one too

Comment (yitzchak) [reply README.md]: How about "The first is the regular release cycle package..."

**Comment by Cogitri** [comment]
Merged, thanks

**Pull request merged**